### ISSUE TITLE
Negative Percentage produces wrong result

### ISSUE THREAD

**Issue by maennchen** [elixir-cldr/cldr_numbers#11]
## Test

```elixir
test "percent" do
  assert TestBackend.Cldr.Number.to_string(0.1, format: :percent) ==
          {:ok, "10%"}
  assert TestBackend.Cldr.Number.to_string(-0.1, format: :percent) ==
          {:ok, "-10%"}
end
```

## Expected

Test Passes

## Result

```
  1) test percent (Number.Format.Test)
     test.exs
     Assertion with == failed
     code:  assert TestBackend.Cldr.Number.to_string(-0.1, format: :percent) == {:ok, "-10%"}
     left:  {:ok, "-10"}
     right: {:ok, "-10%"}
     stacktrace:
       test/number/number_format_test.exs:109: (test)
```

**Comment by kipcole9** [comment]
Thanks for the report. Turns out the issue was broader than just negative percentages so it was a good find.

Updated on master branch and I will release in a couple of hours to hex (I want to add some additional tests first).

**Comment by kipcole9** [comment]
Released on hex as [version 2.10.0](https://hex.pm/packages/ex_cldr_numbers/2.10.0)

**Comment by maennchen** [comment]
@kipcole9 Works like a charm, thanks :)
